src/main/java/io/redisearch/aggregation/AggregationRequest.java

```diff
@@ -33,8 +35,15 @@ public AggregationRequest() {
         this("*");
     }
 
-    public AggregationRequest load(String ...fields) {
+    public AggregationRequest load(String... fields) {
         load.addAll(Arrays.asList(fields));
+        loadFieldNames.clear();
```

Comment:
@sazzad16 why not remove `load` and use `loadFieldNames` in this case too? Just convert each String to `FieldName`

Reply:
@gkorland done